tmt/schemas/provision/kickstart.yaml

```diff
@@ -0,0 +1,38 @@
+---
+
+#
+# JSON Schema definition for kickstart specification
+#
+# https://tmt.readthedocs.io/en/stable/spec/plans.html#kickstart
+#
+
+$id: /schemas/provision/kickstart
+$schema: http://json-schema.org/draft-07/schema
+
+definitions:
+  kickstart:
+    type: object
+
+    properties:
```

Comment:
If we are here, should we also not add:

https://beaker-project.org/docs/user-guide/customizing-installation.html#kernel-options

`kernel-options`?

Reply:
Yes, we can add that as well

Reply:
I added both. 
@juk I'm just not exactly sure about the post options, what exactly does it mean? Is this something
that corresponds to the `GRUB_CMDLINE_LINUX_DEFAULT` variable in /etc/default/grub?

Reply:
I think it's under "bootloader --append" in kickstart
bootloader {% if kernel_options_post %} --append="{{ kernel_options_post }}
something like this

Reply:
@juk, alright thanks!